pallets/proofs-dealer/src/lib.rs

```diff
@@ -80,11 +79,11 @@ pub mod pallet {
 
         /// The maximum number of challenges that can be made in a single block.
         #[pallet::constant]
-        type MaxChallengesPerBlock: Get<u32> + FullCodec;
+        type MaxChallengesPerBlock: Get<u32>;
 
         /// The maximum number of Providers that can be challenged in block.
         #[pallet::constant]
-        type MaxProvidersChallengedPerBlock: Get<u32> + FullCodec;
+        type MaxProvidersChallengedPerBlock: Get<u32>;
```

Comment:
As I said earlier, I only modify things like this if it breaks the build. I am not able to specify a ConstU32 and would have to implement `FullCodec`, instead of doing that, I didn’t see any reason for the trait to be there.

<img width="976" alt="image" src="https://github.com/Moonsong-Labs/storage-hub/assets/94772640/5da6ae8b-10c0-4890-9cd4-18aac36844e1">